src/command/KeyBindingManager.js

```diff
@@ -404,12 +405,19 @@ define(function (require, exports, module) {
         
         // Ensure that the first letter of the key name is in upper case.
         // i.e. 'a' => 'A' and 'up' => 'Up'
-        if (/^[a-z]/.test(key)) {
+        if (/^[a-z]/i.test(key)) {
```

Comment:
`.toUpperCase()` will no-op on non-letters.

And if you really need to check for valid English letters, you can still do it like this:
```JS
if (/^[a-z]/i.test(key)) {
    key = key.charAt(0).toUpperCase() + key.substr(1).toLowerCase();
}
```